install/olm-manifests/bundle.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1323 @@
+                - apps.openshift.io
+              resources:
+                - deploymentconfigs
+                - deploymentconfigs/scale
+                - deploymentconfigs/status
+                - deploymentconfigs/finalizers
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - delete
+                - patch
+                - update
+            - apiGroups:
+                # OpenShift S2I requirements
+                - build.openshift.io
+              resources:
+                - buildconfigs
+                - buildconfigs/instantiate
+                - builds
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - delete
+                - patch
+                - update
+            - apiGroups:
+                # OpenShift S2I requirements
+                - image.openshift.io
+              resources:
+                - imagestreams
+                - imagestreams/status
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - delete
+                - patch
+                - update
+            - apiGroups:
+                - networking.k8s.io
+              resources:
+                # The cluster operator needs to access and manage network policies to lock down communication between Strimzi components
+                - networkpolicies
+                # The cluster operator needs to access and manage ingresses which allow external access to the services in a cluster
+                - ingresses
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - delete
+                - patch
+                - update
+            - apiGroups:
+                - route.openshift.io
+              resources:
+                # The cluster operator needs to access and manage routes to expose Strimzi components for external access
+                - routes
+                - routes/custom-host
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - delete
+                - patch
+                - update
+            - apiGroups:
+                - policy
+              resources:
+                # The cluster operator needs to access and manage pod disruption budgets this limits the number of concurrent disruptions
+                # that a Strimzi component experiences, allowing for higher availability
+                - poddisruptionbudgets
+              verbs:
+                - get
+                - list
+                - watch
+                - create
+                - delete
+                - patch
+                - update
+          serviceAccountName: strimzi-cluster-operator
+      clusterPermissions:
```

Comment:
Ok this may explain the errors I saw there should be a `- rules:` below `clusterPermissions:`

Reply:
And yep this is template is also missing the `serviceAccountName` entry for the `clusterPermissions`.